decentraland/components/presentational/LinkToEtherscan.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Button, Icon } from "native-base";
+import { showError, openURL } from "@helpers";
+
+const _openEtherscanOf = async address => {
+  const url = `https://etherscan.io/address/${address}`;
```

Comment:
This component could get slightly fancier and support address, contract, tx, etc.

Reply:
Note that we'll want to default this to a testnet for now and make that configurable too

Reply:
> This component could get slightly fancier and support address, contract, tx, etc.

For sure.

Reply:
> Note that we'll want to default this to a testnet for now and make that configurable too

Oh, thanks! I've forgetted that.

Reply:
Feel free to resolve this

Reply:
Okay, Resolving.